python/src/gen3authz/client/arborist/base.py

```diff
@@ -270,7 +270,7 @@ async def auth_mapping(self, username):
         return response.json
 
     @maybe_sync
-    async def auth_request(self, jwt, service, methods, resources):
+    async def auth_request(self, jwt, service, methods, resources, user_id=None):
         """
         Return:
```

Comment:
I know there aren't docstring args documented here, but can we quickly add them or at least a note that providing user_id **overrides** anything passed as jwt?